runtime/src/main/java/org/corfudb/runtime/LRMultiNamespaceListener.java

```diff
@@ -82,6 +87,15 @@ public void onNext(CorfuStreamEntries results) {
      */
     protected abstract void processUpdateInLogEntrySync(Map<TableSchema, List<CorfuStreamEntry>> entries);
 
+    /**
+     *
+     */
+    protected abstract CorfuStoreMetadata.Timestamp getSubscriptionTimestamp();
```

Comment:
I think this should be more like Timestamp performFullSync() or performMultiTableReads(), or initialTablesFullSync()